packages/components/src/Actions/ActionDropdown/__snapshots__/ActionDropdown.snapshot.test.js.snap

```diff
@@ -1,1865 +1,468 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ActionDropdown should render "no option" item when items array is empty 1`] = `
-<DropdownMenu
-  bsClass="dropdown-menu"
-  bsRole="menu"
-  key=".1"
-  labelledBy="dropdown-id"
-  onClose={[Function]}
-  onSelect={[Function]}
-  pullRight={false}
+<ul role="menu"
```

Comment:
Sorry for this big diff, but now we print the html instead of the wrappers.
Reading the file will be way easier than to read the  diff